simphony_ui/liggghts_model.py

```diff
@@ -14,48 +15,184 @@ class LiggghtsModel(HasStrictTraits):
     #: The input file used for liggghts.
     input_file = File()
 
-    boundary_condition_x = Enum('periodic', 'fixed', 'shrink-wrapped')
-    boundary_condition_y = Enum('periodic', 'fixed', 'shrink-wrapped')
-    boundary_condition_z = Enum('periodic', 'fixed', 'shrink-wrapped')
+    # Boundary conditions parameters
+    x_wall = Enum('periodic', 'fixed')
```

Comment:
I agree, I thought about prefixing every parameters with 'bc' 'cm' or 'sp'. What do you think?